wallet/lib/pages/home/easel_section/no_easel_art_work.dart

```diff
@@ -17,45 +21,80 @@ class NoEaselArtWork extends StatelessWidget {
     context.watch<HomeProvider>();
 
     return Column(
-      mainAxisAlignment: MainAxisAlignment.center,
-
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      crossAxisAlignment: CrossAxisAlignment.start,
       children: [
-        SizedBox(
-          height: 40.h,
-        ),
         Padding(
-          padding: EdgeInsets.symmetric(
-            horizontal: 20.w,
-          ),
+          padding: EdgeInsets.symmetric(horizontal: 25.w,),
           child: Text(
             LocaleKeys.creation_list_empty_text.tr(),
-            style: TextStyle(fontSize: 15.sp, fontWeight: FontWeight.w700, color: AppColors.kBlue),
-            textAlign: TextAlign.center,
+            style: TextStyle(fontSize: 13.sp, fontWeight: FontWeight.w600, color: AppColors.kGreyColor),
+            textAlign: TextAlign.start,
           ),
         ),
+        const Spacer(),
+        Padding(
+          padding: EdgeInsets.symmetric(horizontal: 25.w,),
+          child: FutureBuilder<Column>(
+            future: _getButtonForEasel(context: context),
+            builder: (context,snapshot){
+              return snapshot.data??const SizedBox();
+            },
+          ),
+        ),
+        SizedBox(height: 50.h,),
+      ],
+    );
+  }
+
+  Future<Column> _getButtonForEasel({required BuildContext context})async{
+    final isAndroidDevice = Theme.of(context).platform == TargetPlatform.android;
+
+    final easelAppInstallLink = isAndroidDevice ? kAndroidEaselInstallLink : kIOSEaselInstallLink;
```

Comment:
These things should be assign on initState otherwise they will be assigned again and again.